extensions/expression-language/impl/expression/Expression.h

```diff
@@ -32,9 +32,9 @@
 #undef EXPRESSION_LANGUAGE_USE_DATE
 #endif
 
-#include <Value.h>
-#include <core/FlowFile.h>
-#include <core/VariableRegistry.h>
+#include "../../common/Value.h"
```

Comment:
I think this relative path was added by way of the IDE's auto rename. should fix.